langchain/src/agents/executor.ts

```diff
@@ -393,10 +394,13 @@ export class AgentExecutor extends BaseChain<ChainValues, AgentExecutorOutput> {
     let agent: BaseSingleActionAgent | BaseMultiActionAgent;
     let returnOnlyOutputs = true;
     if (Runnable.isRunnable(input.agent)) {
-      agent = new RunnableAgent({ runnable: input.agent });
+      agent = new RunnableMultiActionAgent({ runnable: input.agent });
       // TODO: Update BaseChain implementation on breaking change
       returnOnlyOutputs = false;
     } else {
+      if (isRunnableAgent(input.agent)) {
```

Comment:
More of a question for @jacoblee93 but why don't we just add `lc_runnable` to the runnable agent classes? That way we can use the `Runnable.isRunnable()` method

Reply:
Because they're not runnable